Pages/Hotels/HotelHomePage.js

```diff
@@ -135,6 +147,96 @@ async isRemoveChildBttnEnabled() {
     return removeChildBttn.isEnabled();
 }
 
+async selectEspañol() {
+    const langDropDown = await $(this.languageBoxLocator);
+    await langDropDown.selectByIndex(1);
+    await browser.pause(2000);
+}
+
+async selectEnglish() {
+    const langDropDown = await $(this.languageBoxLocator2);
+    await langDropDown.selectByIndex(0);
+    await browser.pause(2000);
+}
+
+async clickEnglishLang() {
+    const clickEnglish = await $(this.enlishLangLocator);
+    await clickEnglish.click();
+    await browser.pause(2000);
+}
+
+async clickSaveBttn() {
+    const saveBttn = await $(this.saveBttnLocator);
+    await saveBttn.click();
+    await browser.pause(2000);
+}
+
+async clickGuardarBttn() {
+    const guardarBttn = await $(this.guardarBttnLocator);
+    await guardarBttn.click();
+    await browser.pause(2000);
+}
+async isEspañolLangDisplayed() {
+    const isEspañolLangDisplaying = await $(this.españolLangLocator);
+    return isEspañolLangDisplaying.isDisplayed()
+    expect(isEspañolLangDisplaying, 'Español is NOT displayed').to.be.true;
+}
+
+async isEnglishLangDisplayed() {
+    const isEnglishLangDisplaying = await $(this.enlishLangLocator);
+    return isEnglishLangDisplaying.isDisplayed()
+    expect(isEnglishLangDisplaying, 'English is NOT displayed').to.be.true;
+}
+async clickEspañolLang() {
+    const clickEspañol = await $(this.españolLangLocator);
+    await clickEspañol.click();
+    await browser.pause(2000);
+}
+
+async clickListYourProperty() {
+    const listYourProperty = await $(this.listYourPropertyLocator);
+    await listYourProperty.click();
+    await browser.pause(2000);
+}
+
+async switchingToPropertyHandle() {
+    const connectedHandle =await browser.getWindowHandle();
+
+    const allHandles = await browser.getWindowHandles();
+    for (const handle of allHandles) {
+        await browser.switchToWindow(handle);
+        const currentUrl = await browser.getUrl();
+        console.log(`current url -> ${currentUrl}\n\n`);
+        if (currentUrl.localeCompare('https://apps.expediapartnercentral.com/en_US/list?utm_medium=referral&utm_source=HCOM_US-en_US&utm_campaign=HomePage&utm_contentewd=pwa-header-btn&siteId=300000001&tpid=3001&eapid=1&langId=1033') === 0) {
+            return handle;
```

Comment:
Pass `https://apps.expediapartnercentral.com/en_US/list?utm_medium=referral&utm_source=HCOM_US-en_US&utm_campaign=HomePage&utm_contentewd=pwa-header-btn&siteId=300000001&tpid=3001&eapid=1&langId=1033` as parameter. then you'll be able to use the same function for several places.
Instead of localCompare you can also use contains() function, then you don't have to pass whole url.